uwblueprint/building-up-client
# Login/Register Component

## Description
Tickets:
[Signup UI](https://www.notion.so/uwblueprintexecs/Sign-up-Frontend-UI-285cd83e4ac04457882e80cd9b44cf26)
[Login UI](https://www.notion.so/uwblueprintexecs/Login-Frontend-UI-fc6e23155665482db710f5091a588440)
[Combining into one](https://www.notion.so/uwblueprintexecs/Combine-login-and-sign-up-on-one-page-7e7d9406f94b4afaaf56b1928dbe8667)

Built the login/signup component so they are both on the same route. Forgot password link is not yet set up and error states are not implemented in these tickets, but login/register should work.

## Testing

- Pull this branch and run the app locally
- Navigate to [http://localhost:3000/login](http://localhost:3000/login) and make sure the UI matches/behaviour is correct

## Screenshots
### Desktop
![2021-02-28 14 28 35](https://user-images.githubusercontent.com/44102730/109430982-05109d00-79d2-11eb-962a-4bba7dbeaaae.gif)

### Tablet
<img width="386" alt="Screen Shot 2021-02-28 at 2 36 06 PM" src="https://user-images.githubusercontent.com/44102730/109431016-51f47380-79d2-11eb-9e0d-717f39291b01.png">

### Mobile
<img width="355" alt="Screen Shot 2021-02-28 at 2 36 37 PM" src="https://user-images.githubusercontent.com/44102730/109431025-5fa9f900-79d2-11eb-8faa-814fecd01641.png">
